src/service/resources-impl.js

```diff
@@ -1329,7 +1329,7 @@ export class Resources {
       }
       // Nothing to do.
       if (opt_callback) {
-        opt_callback(/* hasSizeChanged */false);
+        opt_callback(/* success */ true);
```

Comment:
It's from https://github.com/ampproject/amphtml/blob/be8c8c2526e0a448e9f6c6eb0adbcc98391c8099/src/service/resources-impl.js#L615.